lib/galaxy/tools/cache.py

```diff
@@ -138,66 +132,29 @@ class ToolShedRepositoryCache(object):
 
     def __init__(self, app):
         self.app = app
-        self.cache = local()
+        self.local_repositories = []
```

Comment:
Can you add comment about `self.local_repositories` clarifying it stores tools that are not in the install database?